docs/usage.md

```diff
@@ -0,0 +1,161 @@
+# Usage Guide
+
+## Basic Client Usage
+
+### Asynchronous Client
+
+```python
+from your_generated_code.eliza_connecpy import ElizaServiceClient
+from your_generated_code import eliza_pb2
+
+async def main():
+    async with ElizaServiceClient("https://demo.connectrpc.com") as eliza_client:
+        # Unary responses: await and get the response message back
+        response = await eliza_client.say(eliza_pb2.SayRequest(sentence="Hello, Eliza!"))
+        print(f"  Eliza says: {response.sentence}")
+
+        # Streaming responses: use async for to iterate over messages in the stream
+        req = eliza_pb2.IntroduceRequest(name="Henry")
+        async for response in eliza_client.introduce(req):
+            print(f"   Eliza: {response.sentence}")
+
+        # Streaming requests: send an iterator, get a single message
+        async def pontificate_requests():
+            yield eliza_pb2.PontificateRequest(sentence="I have many things on my mind.")
+            yield eliza_pb2.PontificateRequest(sentence="But I will save them for later.")
```

Comment:
PontificateRequest seems not to be included in eliza.proto (but maybe it's ok for just documentation?)